resources/views/student/exam.blade.php

```diff
@@ -161,5 +166,32 @@
     document.getElementById("mathLive").appendChild(mathField)
 
 
+     // -------------------------- Listeners -----------------------------------------
+     examFormButton.addEventListener('click', () => { examFormElement.submit() });
+
+    //-------------------------------- Leave tab, page is not visible -------------------------
+    document.addEventListener('visibilitychange', onVisibilityChange);
+
+
+    //-------------------------------- Functions -------------------------
+    function onVisibilityChange(e) {
+        //Ak leavne tab
+        if(document.hidden) {
+            changeStatusOfExam('left');
+        } else {
+            changeStatusOfExam('writing');
+        }
+    }
```

Comment:
Tu ale neviem ci nebudeme musiet aj nieco robit specialne ked opusti tab, musim precitat zadanie

Reply:
Nie nie, ja som cital doc a tam je len ze notifikovat ucitela ak leavne tab takze tam sa posle len AJAXom zmena statusu a a v admine ucitela pekne cez socket potiahnem real time tu zmenu a bude to hotove.

Reply:
Tak parada